doc/whats_new.rst

```diff
@@ -144,6 +144,18 @@ Enhancements
    - The :func: `ignore_warnings` now accept a category argument to ignore only
      the warnings of a specified type. By `Thierry Guillemot`_.
 
+   - The `grid_scores_` attribute of :class:`model_selection.GridSearchCV` and
+     :class:`model_selection.RandomizedSearchCV` is deprecated in favor of the
+     attribute, `results_`, which is a dict of numpy (masked) arrays. This dict
+     can be easily imported into pandas as a dataframe for exploring the search
+     results. By `Raghav RV`_.
+
+   - The :class:`model_selection.GridSearchCV` and
+     :class:`model_selection.RandomizedSearchCV` now have a new attribute,
+     `candidate_params`, which is a list of parameter dicts (settings) for all
```

Comment:
I would suggest to keep it for lazy users. It is easier to quickly get the number of parameter settings (without pandas) if not anything else.